queries/mutation.gql

```diff
@@ -1,27 +1,23 @@
 mutation installBundle (
     $activeConnectors: [ActiveConnectorInput!]!=[],
     $administrativeUserAccountProperties: [AdministrativeUserAccountPropertyInput!]!=[],
-    $backgroundTaskPolicies:[BackgroundTaskPolicyInput!]!=[],
```

Comment:
I hope, you pulled this content from UneasyRooster!

Reply:
updated with uneasyrooster file